examples/framework/preferences/conditionalAdjusters-singlePanel/jQueryUI.html

```diff
@@ -0,0 +1,99 @@
+        <script type="text/javascript" src="../../../../src/lib/fastXmlPull/js/fastXmlPull.js"></script>
+        <script type="text/javascript" src="../../../../src/framework/renderer/js/fluidParser.js"></script>
+        <script type="text/javascript" src="../../../../src/framework/renderer/js/fluidRenderer.js"></script>
+        <script type="text/javascript" src="../../../../src/framework/renderer/js/RendererUtilities.js"></script>
+
+        <script type="text/javascript" src="../../../../src/framework/preferences/js/URLUtilities.js"></script>
+        <script type="text/javascript" src="../../../../src/framework/preferences/js/Store.js"></script>
+        <script type="text/javascript" src="../../../../src/components/textfieldSlider/js/TextfieldSlider.js"></script>
+        <script type="text/javascript" src="../../../../src/framework/preferences/js/UIEnhancer.js"></script>
+        <script type="text/javascript" src="../../../../src/framework/preferences/js/PrefsEditor.js"></script>
+        <script type="text/javascript" src="../../../../src/framework/preferences/js/Panels.js"></script>
+        <script type="text/javascript" src="../../../../src/framework/preferences/js/Enactors.js"></script>
+        <script type="text/javascript" src="../../../../src/framework/preferences/js/SeparatedPanelPrefsEditor.js"></script>
+        <script type="text/javascript" src="../../../../src/components/slidingPanel/js/SlidingPanel.js"></script>
+        <script type="text/javascript" src="../../../../src/components/tableOfContents/js/TableOfContents.js"></script>
+        <script type="text/javascript" src="../../../../src/framework/preferences/js/PrimaryBuilder.js"></script>
+        <script type="text/javascript" src="../../../../src/framework/preferences/js/AuxBuilder.js"></script>
+        <script type="text/javascript" src="../../../../src/framework/preferences/js/StarterSchemas.js"></script>
+        <script type="text/javascript" src="../../../../src/framework/preferences/js/Builder.js"></script>
+
+        <script type="text/javascript" src="../shared/js/primarySchema.js"></script>
+        <script type="text/javascript" src="../shared/js/utilities.js"></script>
+        <script type="text/javascript" src="js/auxiliarySchema.js"></script>
+        <script type="text/javascript" src="../shared/js/panels.js"></script>
+        <script type="text/javascript" src="js/panels.js"></script>
+        <script type="text/javascript" src="../shared/js/enactors.js"></script>
+        <script type="text/javascript" src="js/conditional.js"></script>
+    </head>
+
+    <body class="example-theme fl-focus">
+        <script type="text/javascript">
+
+            fluid.contextAware.makeChecks({
+                "fluid.prefsWidgetType": {
+                    value: "jQueryUI"
+                }
+            });
+
+            $(document).ready(function () {
+                example.initPrefsEditor(".flc-prefsEditor-separatedPanel");
+            })
```

Comment:
Rather than have this `$(document).ready()` call, we should move these to a script tag in the appropriate position in the DOM after all it's required markup has loaded.